### ISSUE TITLE
The way to support virtual keyboard in win10

### ISSUE THREAD

**Issue by chaome** [tishion/QCefView#83]
1. QCefViewBrowserHandler.h
  Add override method to return this as focus handler.
``` cpp
  virtual CefRefPtr<CefFocusHandler> GetFocusHandler() override
  {
    return this;
  }
```

2. QCefViewBrowserHandler.h/cpp
Add override method to handle focus event, set focus when browser need focus.
``` cpp
virtual void OnGotFocus(CefRefPtr<CefBrowser> browser) override;

void
QCefViewBrowserHandler::OnGotFocus(CefRefPtr<CefBrowser> browser)
{
  CEF_REQUIRE_UI_THREAD();

  auto h = main_browser_->GetHost();
  if (h != nullptr) {
    h->SetFocus(true);
  }
}
```

3. QCefViewBrowserApp.cpp
Add command to disable usb keyboard detect;

``` cpp
QCefViewBrowserApp::OnBeforeCommandLineProcessing(...)
{
....

command_line->AppendSwitchWithValue("disable-usb-keyboard-detect", "1");
}

```

Tested: Qt5.12.10/msvc2017/Chrome 87.0.4280.88/Windows10 Enterprise

**Comment by tishion** [comment]
can you create a pull request and submit ?

**Comment by tishion** [comment]
Fixed by PR #84
